piximi-documentation/classification.md

```diff
@@ -0,0 +1 @@
+# Classification
```

Comment:
Could you add stub "coming soon" or other text to the stub pages, please and thank you?